apps/frontend/src/pages/friends/friends.tsx

```diff
@@ -0,0 +1,16 @@
+import { DEFAULT_FRIENDS_DATA } from "~/libs/constants/constants.js";
+
+import { Friend } from "./components/components.js";
+import styles from "./styles.module.css";
+
+const Friends: React.FC = () => {
+	return (
+		<ul className={styles["wrapper"]}>
+			{DEFAULT_FRIENDS_DATA.map((friend) => (
+				<Friend friend={friend} key={friend.id} />
```

Comment:
```suggestion
    <li>
				<Friend friend={friend} key={friend.id} />
				</li>
```
inside an `ul` element on an `li` element is allowed